docs/topics/Permissions.md

```diff
@@ -79,6 +79,7 @@ Below is a table of all current permissions, their integer values in hexadecimal
 | CREATE_EVENTS                          | `0x0000100000000000` `(1 << 44)` | Allows for creating scheduled events, and editing and deleting those created by the current user. Not yet available to developers, [see changelog](#DOCS_CHANGE_LOG/clarification-on-permission-splits-for-expressions-and-events).                        | V, S         |
 | USE_EXTERNAL_SOUNDS                    | `0x0000200000000000` `(1 << 45)` | Allows the usage of custom soundboard sounds from other servers                                                                                                                                                                                            | V            |
 | SEND_VOICE_MESSAGES                    | `0x0000400000000000` `(1 << 46)` | Allows sending voice messages                                                                                                                                                                                                                              | T, V, S      |
+| SEND_POLLS                             | `0x0002000000000000` `(1 << 49)` | Allows sending polls                                                                                                                                                                                                                                       | T, V, S      |
```

Comment:
@tpcstld  Should this be `CREATE_POLLS` to match what the client shows? 
![img](https://i.imgur.com/RfjFZ1S.png)

Reply:
No, this is what we are calling the permission internally, we try to use SEND_ or USE_ suffix.